scripts/semantic-release/sr-build-package.ts

```diff
@@ -429,6 +430,11 @@ export function getContractAddress<T extends EcoChainIdsEnv>(
 ): Hex {
   return EcoProtocolAddresses[chainId][contractName] as Hex;
 }
+
+/**
+ * The multi-chain address type for the eco protocol.
+ */
+export type EcoAddress = EvmAddress | SvmAddress;
```

Comment:
```suggestion
export type Address = EvmAddress | SvmAddress;
```
EcoAddress is kinda weird.